src/ui/osx/TogglDesktop/LoginViewController.m

```diff
@@ -192,29 +207,49 @@ - (void)textFieldClicked:(id)sender
 
 - (void)changeTabView:(TabViewType)type
 {
-	[[NSNotificationCenter defaultCenter] postNotificationOnMainThread:kHideDisplayError
-																object:nil];
-	self.currentTab = type;
+    [self changeTabView:type hideErrorMessage:YES];
+}
 
-	// Focus on email when changing mode
-	[self.view.window makeFirstResponder:self.email];
+- (void)changeTabView:(TabViewType)type hideErrorMessage:(BOOL) hideErrorMessage
+{
+    if (hideErrorMessage) {
+        [[NSNotificationCenter defaultCenter] postNotificationOnMainThread:kHideDisplayError
+                                                                    object:nil];
+    }
 
-	switch (type)
-	{
-		case TabViewTypeLogin :
+    self.currentTab = type;
+    [self showLoaderView:NO];
+
+    // Focus on email when changing mode
+    [self.view.window makeFirstResponder:self.email];
+
+    switch (type)
+    {
+        case TabViewTypeLogin :
             self.appleGoogleGroupViewTop.constant = kLoginAppleViewTop;
             self.containerViewHeight.constant = kLoginContainerHeight;
             self.signUpGroupView.hidden = YES;
-            self.appleBtn.title = @" Log in with Apple";
+            self.appleBtn.title = @" Sign in with Apple";
             self.googleBtn.title = @" Log in with Google";
```

Comment:
My personal opinion is that we should rename "Log in with Google" to "Sign in with Google". "Sign in" also seems to be used by Google itself https://developers.google.com/identity/sign-in/web/sign-in.

Reply:
It's our designer's decision. We changed "Sign in with Apple" but keep "Log in with Google", but it's reasonable to make it consistent 👍